src/UserApp.ts

```diff
@@ -0,0 +1,173 @@
+import { DivsStructure } from "../utils/HtmlStructures"
+import { $, render, toLCaseIncludes, toArr } from "./variables/functions"
+
+
+// Enum with members representing buttons on the top of the app
+export enum CurrentButton {
+    Search,
+    Add,
+    Remove,
+    ShowAll
+}
+// User class
+export class User {
+    name: string
+    surname: string
+    dateOfBirth: number
+
+    constructor( name: string, surname: string, dateOfBirth: number ) {
+        this.name = name
+        this.surname = surname
+        this.dateOfBirth = dateOfBirth
+    }
+
+}
+// State class
+class UserAppState {
+    // Current button type
+    btnState: CurrentButton
+    // Array of found users
+    foundUsers: Array<User> = []
+    // Check if submit button clicked
+    ifSbmtClicked: boolean = false
```

Comment:
For booleans indicating whether something is true or not, there are two naming conventions I can recommend, please choose one and follow it:
1) Question-based: `isSubmitClicked`, `isSendingRequest` etc
2) A little older version of (1), used in e.g. Unreal codebase: `bIsSubmitClicked`, `bIsSendingRequest` etc.
 
The idea with (2) is to have "b" as first letter of any boolean. The benefit of this is that if you see ANY variable with "b" at the start - you automatically know it is boolean. I like it, but you can follow (1) as well if you find (2) tedious.